addons/mail/static/src/discuss/core/public_web/store_service_patch.js

```diff
@@ -8,7 +8,7 @@ import { patch } from "@web/core/utils/patch";
 const StorePatch = {
     setup() {
         super.setup(...arguments);
-        this.channels = this.makeCachedFetchData("channels_as_member");
+        this.channels = this.makeCachedFetchData("channels_as_member", { readonly: false });
```

Comment:
This is not the right place to add it.
Fetching is related to message, not channel data.